src/main/scala/io/treeverse/examples/List.scala

```diff
@@ -14,17 +13,13 @@ object List extends App {
       .scanLeft("")((a, b: String) => a + "/" + b)
 
   override def main(args: Array[String]) {
-    if (args.length != 1) {
-      Console.err.println("Usage: ... s3://path/to/metarange")
-      System.exit(1)
-    }
-
-    val path = args(0)
-
-    val sc = new SparkContext(new SparkConf().setMaster("local").setAppName("I can list"))
-
-    val files = sc.newAPIHadoopFile[Array[Byte], WithIdentifier[Catalog.Entry], LakeFSInputFormat](path)
-
+    val spark = SparkSession.builder().master("local").appName("I can list")
+      .config("spark.hadoop.lakefs.api.url", "http://localhost:8000/api/v1/")
+      .config("spark.hadoop.lakefs.api.access_key", "AKIAIOSFODNN7EXAMPLE")
```

Comment:
These are never necessary if you set environment variables (but not `AWS_PROFILE`).  Having them here *overrides* environment variables, so I don't think we want them here.
Also doesn't having a string `AKIA...` trigger some security watchers that look for leaked secrets?

Reply:
These are lakeFS credentials, they don't have environment variables yet.

I think the ones that end with EXAMPLE are spared

Reply:
Oh right, I don't know how to read.  Let's just write the secret key literally as `"..."`?